package.json

```diff
@@ -39,8 +39,8 @@
     "js-yaml": "^3.14.0"
   },
   "dependencies": {
-    "@startbahn/startrail": "1.8.0-beta-3",
+    "@startbahn/startrail": "1.7.0-rc1",
```

Comment:
Do we need to change this when it's go to deploy 1.7.0 ?

Reply:
yes i think once tests pass and it's in master the startail npm package can be tagged with 1.7.0. do we have a procedure for this from previous releases?

Reply:
> yes i think once tests pass and it's in master the startail npm package can be tagged with 1.7.0. do we have a procedure for this from previous releases?

Pleasure if I could have a better understanding of this.